libkbfs/config_local.go

```diff
@@ -208,6 +209,15 @@ func MakeLocalUsers(users []libkb.NormalizedUsername) []LocalUser {
 	return localUsers
 }
 
+func getDefaultCleanBlockCacheCapacity() uint64 {
+	vmstat, err := mem.VirtualMemory()
+	if err == nil {
+		// By default,  we try to cap at 1/8 of available memory
```

Comment:
Super minor nits: looks like you have two spaces after "By default,", when you only need one.  Also. the comment should end in a period.